mina-archive/templates/configmap-runtime-config.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.archive.runtimeConfig }}
+{{- if .Values.archive.initFromDump }}
```

Comment:
In deployment template you are checking if `.Values.archive.initFromDump.enabled` and here you're checking `.Values.archive.initFromDump`. Is that intentional?

Reply:
It is a copy/paste issue. Thank you for pointing that out